src/relay/backend/graph_plan_memory.cc

```diff
@@ -376,12 +385,12 @@ class StorageAllocator : public StorageAllocaBaseVisitor {
       return fn->HasNonzeroAttr(attr::kReshapeOnly);
     }
 
-    if (call->attrs.defined()) {
-      if (auto tir_call_attrs = call->attrs.as<TIRCallAttrs>()) {
-        Map<String, ObjectRef> metadata = tir_call_attrs->metadata;
-        return metadata.count(attr::kReshapeOnly) &&
-               (Downcast<tvm::Integer>(metadata[attr::kReshapeOnly])->value == 1);
-      }
+    if (call->op == CallLoweredOp()) {
+      auto call_lowered_attrs = call->attrs.as<CallLoweredAttrs>();
```

Comment:
nit: just calling your Extract helper would be easier assuming it gave you the attrs in the returned struct (as per other comment).